assets/styles/_styles.scss

```diff
@@ -0,0 +1,110 @@
+/* ---------- VARIABLES ---------- */
+:root {
+  --box-shadow: 0px 6px 9px 0px rgb(0 0 0 / 15%);
+  --box-shadow-nav: rgb(0 0 0 / 22%) 0px 2 18px 0px;
+  --text-color-black: black;
+  --text-color-white: white;
+  --background: #F1FAEE;
+  --background-blue: #1D3557;
+}
+
+/* ---------- FONTS ---------- */
+@font-face {
+  font-family: 'Happy Monkey';
+  src: url("../fonts/HappyMonkey-Regular.ttf");
+}
+
+/* ---------- GLOBAL POLICIES ---------- */
+
+* {
+  box-sizing: border-box;
+  font-family: 'Happy Monkey', serif;
+}
+
+html {
+  font-size: 10px;
```

Comment:
Why do you fix the font size on the html, and you do the same thing in the body ?